docs/api/auto-updater.md

```diff
@@ -97,23 +97,33 @@ When this API is called, the `before-quit` event is not emitted before all windo
 
 The `autoUpdater` object has the following methods:
 
-### `autoUpdater.setFeedURL(options)`
+### `autoUpdater.initialize(options)`
 
 * `options` Object
   * `url` String
   * `headers` Object (optional) _macOS_ - HTTP request headers.
   * `serverType` String (optional) _macOS_ - Either `json` or `default`, see the [Squirrel.Mac][squirrel-mac]
     README for more information.
 
-Sets the `url` and initialize the auto updater.
+Sets the `url` and initializes the auto updater.
 
-### `autoUpdater.getFeedURL()`
+### `autoUpdater.setFeedURL(options)`  _(Deprecated)_
```

Comment:
Can you update this `Deprecated` styling to match style previously used for other property deprecations?